lab-ken/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "lab-ken",
+  "version": "1.0.0",
+  "description": "first lab for JS401",
+  "main": "hello.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
```

Comment:
Featured tasks in Lab 2 said to add scripts for "test" and "lint." Refer to the package.json in Tuesday's example code when Duncan went over Gulp to see what this means. 

This gives me, and anyone else who pulls your files down, to just type the command "npm run test" or "npm run lint" in the terminal order to just run your tests or lint. It allows someone to not have to install either the mocha or eslint dependencies on their own local machine.